beta/src/components/MDX/MDXComponents.tsx

```diff
@@ -234,8 +234,8 @@ function IllustrationBlock({
       ) : null}
       {sequential ? (
         <ol className="mdx-illustration-block flex">
-          {images.map((x: any) => (
-            <li className="flex-1">{x}</li>
+          {images.map((x: any, i :number) => (
```

Comment:
nit: `i: number`, not `i :number`. why didn't Prettier catch this?

Reply:
🤔  Not sure how this was missed.

Reply:
Changed

Reply:
Yea but can we figure out why Prettier didn't fail CI if it mismatched.

Reply:
Maybe Prettier only runs ci for non-beta code? Then let's fix that.

Reply:
It seems like we have the `prettier` command but haven't run it 🤔 . Should we add the same to pre commit as well ?

Reply:
Yes. Also it should fail CI if it _doesn't_ pass. See `ci-check` task in package.json. We need to run this on CI if it doesn't already.

Reply:
So we already have a workflow for checking in the legacy site. Shall I add the ci-check to run there itself for the beta directory too ? Or shall I add a seperate workflow for it ?

Reply:
https://github.com/reactjs/reactjs.org/pull/4100 @gaearon  Since there were lot of file changes as prettier ran,  I've actually raised a seperate PR for that.Now we will run prettier before commits and have a workflow to check if any code is not prettified.